test/server/cards/01_SOR/units/ChopperMetalMenace.spec.ts

```diff
@@ -0,0 +1,63 @@
+describe('Chopper, Metal Menace', function() {
+    integration(function(contextRef) {
+        describe('Chopper\'s On Attack ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['chopper#metal-menace'],
+                        leader: 'hera-syndulla#spectre-two',
+                    },
+                    player2: {
+                        deck: ['battlefield-marine', 'pyke-sentinel', 'underworld-thug', 'the-chaos-of-war', 'volunteer-soldier'],
+                        resources: 5
+                    }
+                });
+            });
+
+            it('should discard a card on attack and not exhaust an enemy resource if the card is not an event; should not have Raid 1', function () {
+                const { context } = contextRef;
+
+                expect(context.player2.deck.length).toBe(5);
+                expect(context.player2.countSpendableResources()).toBe(5);
+
+                context.player1.clickCard(context.chopper);
+                expect(context.player2.base.damage).toBe(1);
+
+                // Check mill and heal
+                expect(context.player2.deck.length).toBe(4);
```

Comment:
Can we confirm that the correct card was milled since this is the first test of this system